src/components/App.js

```diff
@@ -52,6 +53,36 @@ function App() {
                                 ]}
                             />
                         </Route>
+                        <Route path='/product/product-detail-images' exact>
+                            <ProductDetailImages
+                                images={[
```

Comment:
line 58: does images prop will be populated from mongo ? and shouldnt you include maxQuantity?

Reply:
Great point about including `maxQuantity`; I didn't do it here, but I will do it when creating the actual page. I am unsure what you meant by the first question, but the images will be received by making an API call.